src/commands/svc/stop.ts

```diff
@@ -0,0 +1,49 @@
+import { CliUx, Flags } from '@oclif/core'
+import { BundleService } from '../../services/bundle-service'
+import { SvcService } from '../../services/svc-service'
+import { SvcProcessResult, ServiceTypes } from './svc-process'
+
+export default class Stop extends SvcProcessResult {
+  static strict = false
+  static description = 'Stop running enabled auxiliary services'
```

Comment:
Stops

Reply:
I beg to differ. `Stop` is the right word to go with `services` since the subject `services` is a plural

Reply:
I wrote it because all the other command descriptions are using the third-person singular verb. In the sense "This command does this thing".

Reply:
I misunderstood it then. I'll update the other descriptions on other commands as well. They are using the same grammer

Reply:
done